src/core/buffers/representation/representation_buffer.ts

```diff
@@ -197,7 +212,8 @@ export default function RepresentationBuffer<T>({
     terminate$.pipe(take(1), mapTo(true), startWith(false)),
     finishedDownloadQueue$.pipe(startWith(undefined))
   ).pipe(
-    map(function getCurrentStatus([timing, bufferGoal, terminate]) : {
+    withLatestFrom(reporter$.pipe(startWith(undefined))),
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this redundant "undefined". ([typescript:S4623](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4623))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=442&issues=AWgurJA0_MPaBDn0pTR5&open=AWgurJA0_MPaBDn0pTR5)

[ISSUE_KEY]: AWgurJA0_MPaBDn0pTR5